setup.py

```diff
@@ -1,8 +1,53 @@
-from distutils.core import setup
-from Cython.Build import cythonize
-import numpy
-
-setup(
-    ext_modules=cythonize("*.pyx", language="c++"),
-    include_dirs=[numpy.get_include()]
-)
+from setuptools import Extension, find_packages, setup
+
+if __name__ == '__main__':
+  # see https://stackoverflow.com/questions/2379898/make-distutils-look-for-numpy-header-files-in-the-correct-place
```

Comment:
Thanks for the stackoverflow link. I'm a little bothered that it's more than 72 characters long though.

You can probably get away with using `https://stackoverflow.com/a/42163080/2014591`.

Reply:
Oh lovely thank you. I had never used the 'share' button on SO before. TIL!